pandas/tests/test_window.py

```diff
@@ -46,6 +46,27 @@ def win_types_special(request):
     return request.param
 
 
+# Issue 11704: Iteration over a Window
+
+@pytest.fixture
```

Comment:
The fixtures here are a little overkill. Just parametrize a "klass" of `pd.Series` and `pd.DataFrame` (both can be constructed from the values provided).